packages/calling-component-bindings/src/baseSelectors.ts

```diff
@@ -21,8 +28,33 @@ export const getIncomingCallsEnded = (state: CallClientState): IncomingCallState
 
 export const getDeviceManager = (state: CallClientState): DeviceManagerState => state.deviceManager;
 
-export const getCall = (state: CallClientState, props: CallingBaseSelectorProps): CallState | undefined =>
-  state.calls[props.callId];
+export const getCallExists = (state: CallClientState, props: CallingBaseSelectorProps): boolean =>
+  !!state.calls[props.callId];
+
+export const getRemoteParticipants = (
+  state: CallClientState,
+  props: CallingBaseSelectorProps
+):
+  | undefined
```

Comment:
Am I reading this wrong? Why is the type `| undefined | {....}` The first element in the union is nothing?

Reply:
It is beautifier forcing me to switch the line and add another | there

(Looks like it stands for switch a line for nothing)